.github/workflows/test.yml

```
@@ -47,7 +47,7 @@ jobs:
           npm ci
       - name: Synthesize CDK template - Sandbox
         run: |
-          cdk synth -c "atat:EnvironmentId=CiTest" -c "atat:Sandbox=true"
+          cdk synth -c "atat:EnvironmentId=CiTest" -c "atat:Sandbox=true" -c "atat:VersionControlBranch=repoBranch"
```

Make the default value of the context key `develop` so it works similarly by default to the way it did before.

```suggestion
          cdk synth -c "atat:EnvironmentId=CiTest" -c "atat:Sandbox=true" -c "atat:VersionControlBranch=develop"
```

---

done